source/analysis/AM2DNormalizationAB.cpp

```diff
@@ -30,7 +32,9 @@ AM2DNormalizationAB::AM2DNormalizationAB(const QString &outputName, QObject *par
 	canAnalyze_ = false;
 	dataName_ = "";
 	normalizationName_ = "";
-    cacheUpdateRequired_ = false;
+	cacheUpdateRequired_ = false;
```

Comment:
They should all be false to start with I believe.  You should only say you have to update the data after you have sources to get data from.  I can look into these again to make sure they are consistent.  Good catch!